imap_data_access/cli.py

```diff
@@ -439,7 +439,7 @@ def main():  # noqa: PLR0915
         args.func(args)
     except Exception as e:
         # Make sure we are exiting with non-zero exit code and printing the message
-        parser.exit(status=1, message=str(e) + "\n")
+        parser.exit(status=1, message=f"{e!r}\n")
```

Comment:
This gets the `__repr__` of the error message, which means we now get `FileNotFoundError(path/to/file)` instead of just `path/to/file`

Reply:
Nice, thank you!